src/main/kotlin/org/b12x/gfe/GSG.kt

```diff
@@ -2,16 +2,16 @@ package org.b12x.gfe
 
 import javafx.stage.Stage
 import org.b12x.gfe.core.view.MainView
-import org.b12x.gfe.plugins.gfesearch.view.GfeSearchLayoutData
 import tornadofx.App
-import tornadofx.Stylesheet.Companion.label
 
 class GSG: App(MainView::class, Styles::class) {
+
+    val gsgWidth = 1150.0
```

Comment:
Make `private` and or declare as `const`? https://developer.android.com/kotlin/style-guide#constant_names